src/services/git/GitServiceUtils.ts

```diff
@@ -27,36 +29,67 @@ export class GitServiceUtils {
     };
   };
 
-  static getPath = (service: GitService, path: string, branch = 'master') => {
-    switch (service) {
-      case GitService.github:
-        return `/tree/${branch}${path}`;
-      case GitService.bitbucket:
-        return `/src/${branch}${path}`;
-      case GitService.gitlab:
-        return `/tree/${branch}${path}`;
-
-      default:
-        return assertNever(service);
+  static getPath = (componentPath: string, branch = 'master', serviceType: ServiceType) => {
+    if (serviceType) {
```

Comment:
I am not sure if you need this if condition since the `serviceType` is already a required argument that cannot be undefined or null.

Reply:
You're right I've just forgotten to remove it :) Nice catch!